plugins/kubernetes/config/initializers/watchers.rb

```diff
@@ -0,0 +1,65 @@
+require 'celluloid/current'
+require 'celluloid/io'
+require 'kubeclient'
+require 'http'
+require 'json'
+require 'logger'
+
+# monkey patching kubeclient to use non-blocking IO in watch streams
```

Comment:
This is the original for reference: https://github.com/abonas/kubeclient/blob/master/lib/kubeclient/watch_stream.rb